CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 2.22.0-rc.0 / 2020-10-07
+
+As announced in the 2.21.0 release notes, the experimental gRPC API v2 has been
+removed.
+
+* [CHANGE] web: Remove APIv2. #7935
+* [ENHANCEMENT] React UI: Implement missing TSDB head stats section. #7876
+* [ENHANCEMENT] UI: Add Collapse all button. #6957
+* [ENHANCEMENT] UI: Clarify alert state toggle via checkbox icon. #7936
+* [ENHANCEMENT] Add `rule_group_last_evaluation_samples` and `prometheus_tsdb_data_replay_duration_seconds` metrics. #7737 #7977
+* [ENHANCEMENT] Gracefully handle unknown WAL record types. #8004
+* [ENHANCEMENT] Issue a warning for 64 bit systems running 32 bit binaries. #8012
```

Comment:
Is it worth mentioning?

Reply:
I was thinking the same, however incidentally mentioning that 32bit is unwise probably isn't a bad idea so I didn't comment on it.

Reply:
The text also is not correct, it will mention it also on 32 bit systems.